examples/socketmode_handler/socketmode_handler.go

```diff
@@ -56,6 +57,8 @@ func main() {
 
 	// Handle a specific event from EventsAPI
 	socketmodeHandler.HandleEvents(slackevents.AppMention, middlewareAppMentionEvent)
+	socketmodeHandler.HandleEvents(slackevents.Message, middlewareMessageEvent)
```

Comment:
I would prefer if this was an example on its own. That's to make sure that people that don't have this API yet, won't be affected if they want to run this example (basically it will save me answering queries about why this example might not be working).